dist/s_clang-format

```diff
@@ -44,34 +44,50 @@ current_version=`clang-format --version` || exit 1
 
 echo $current_version | grep "version $desired_version" >/dev/null 2>&1
 if test $? -ne 0; then
-	download_clang_format
+    download_clang_format
 fi
 
+exclude=true
 case $# in
 0)
-	# Get all source files that aren't in s_clang-format.list.
-	search=`find bench examples ext src test -name '*.[ch]' -o -name '*.cxx' -o -name '*.cpp'`
-	for f in `cat dist/s_clang-format.list`; do
-		search=`echo "$search" | sed "\#$f#d"`
-	done;;
+    # Get all source files.
+    search=`find bench examples ext src test -name '*.[ch]' -o -name '*.cxx' -o -name '*.cpp'`
```

Comment:
In a previous comment thread you mentioned that our C++ standard doesn't use `*.cxx` file names. Should we remove that from the `find` here for consistency?

Reply:
I requested we keep `.cxx` in [this comment](https://github.com/wiredtiger/wiredtiger/pull/8570#discussion_r1042613695) as there's one straggler file `workgen.cxx` still needing of a rename. Once that's done we can definitely drop this 👍

Reply:
Thanks @ajmorton. Sorry I missed that. I'll resolve this thread.